### DISCUSSION TITLE
Where is nextcloud.log for NextCloud AIO container ?

### DISCUSSION THREAD

**Discussion by IssueFindings** [nextcloud/all-in-one#3824] [Questions] [answered]
Hello,
I need to share date from nextcloud.log with others applications. But I don't know where is located nextcloud.log.
I already search in the directory /var/lib/docker/volumes/nextcloud_aio_mastercontainer but I find nothing :-(

I would like to point out that this is a docker installation with the start command `docker run -d --init --sig-proxy=false --name nextcloud-aio-mastercontainer --restart always --publish 8080:8080 --env NEXTCLOUD_DATADIR="/home/USER/ncdata/data" --env APACHE_PORT=11000 --env APACHE_IP_BINDING=0.0.0.0 --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config --volume /var/run/docker.sock:/var/run/docker.sock:ro nextcloud/all-in-one:latest`

Many thanks for your help. Have a nice day.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#fail2ban
